### PR TITLE
Update a few Regs to correctly refer to "solve" instead of "attempt".

### PR DESCRIPTION
**Pull request by lgarron**
Thanks to Manu Vereecken for spotting these.

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -177,10 +177,10 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 5b2) Any repair to a puzzle must not give the competitor an advantage in solving the puzzle. Penalty: disqualification of the attempt (DNF).
     - 5b3) Permitted repairs:
         - 5b3a) If any parts have fallen out or moved out of place, the competitor may place them back.
-        - 5b3b) If, after repairing the puzzle but before the end of the attempt, the competitor finds that the puzzle is unsolvable, they may disassemble and reassemble a maximum of 4 pieces to make the puzzle solvable.
+        - 5b3b) If a competitor repairs the puzzle but finds that the puzzle is unsolvable later during the solve, they may disassemble and reassemble a maximum of 4 pieces to make the puzzle solvable.
```
Comment (Claster) [inline wca-regulations.md]: Nit: "the competitor" is more common in Regulations.

**Review by Claster — COMMENTED** [review]
Btw, how about clarifying in 2i the time frame when competitors cannot use electronic devices, the same way it is done in 2i2?

2i) While competing, competitors must not use electronics or audio equipment (e.g. cell phones, MP3 players, dictaphones, additional lighting) apart from the Stackmat timer or stopwatch.
->
2i) **From the start of the attempt until the competitor stops the solve, they** must not use electronics or audio equipment (e.g. cell phones, MP3 players, dictaphones, additional lighting) apart from the Stackmat timer or stopwatch.

File: wca-regulations.md (hunk 1 above)
Comment (hanwu85) [reply wca-regulations.md]: I agree, "the competitor" would be more consistent.

**Review by hanwu85 — CHANGES_REQUESTED** [review]
Lucas, could you change the "a competitor" to "the competitor"?  Thanks!

File: wca-regulations.md (hunk 1 above)
Comment (Ivan-Brigidano) [inline wca-regulations.md]:
```suggestion
        - 5b3b) If the competitor repairs the puzzle but finds that the puzzle is unsolvable later during the solve, they may disassemble and reassemble a maximum of 4 pieces to make the puzzle solvable.
```

**Review by Ivan-Brigidano — CHANGES_REQUESTED** [review]

**Comment by lgarron** [comment]
> Lucas, could you change the "a competitor" to "the competitor"? Thanks!

Done, thanks for @Ivan-Brigidano for commenting inline!

**Review by Ivan-Brigidano — APPROVED** [review]

**Review by hanwu85 — APPROVED** [review]
Thanks @lgarron and @Ivan-Brigidano !

**Pull request merged**